includes/discount-actions.php

```diff
@@ -94,6 +94,9 @@ function edd_dcg_create_discount_codes( $data ) {
 	);
 	if ( function_exists( 'edd_add_adjustment' ) ) {
 		$code['scope'] = ! empty( $data['not_global'] ) ? 'not_global' : 'global';
+		if ( isset( $code['expiration'] ) ) {
+			$code['expiration'] = date( 'Y-m-d', strtotime( $code['expiration'] ) . '23:59:59' );
```

Comment:
Very close, but let's put `23:59:59` directly inside the date format like this:

```suggestion
			$code['expiration'] = date( 'Y-m-d 23:59:59', strtotime( $code['expiration'] ) );
```

The first parameter of `date()` normally accepts a format string, but you can also pass in literal numbers as well.